### ISSUE TITLE
Feature request: add snippet for "graphql-request"

### ISSUE THREAD

**Issue by bitjson** [OneGraph/graphiql-code-exporter#34]
Thanks for the fantastic library! I've appreciated it in [Hasura](https://github.com/hasura/graphql-engine/).

It would be great to add support for [`graphql-request`](https://github.com/prisma-labs/graphql-request). Has anyone implemented a custom snippet for `graphql-request`? (I might try to send a PR, just wanted to open an issue first.)

**Comment by sgrove** [comment]
No, that would be great! I was also thinking I should add one for `react-query` at some point, just haven't gotten around to it yet.

We'll be releasing a new version of the exporter that handles subscriptions as well, so maybe keep that in mind if you'd like to provide support for that too in your snippet 😄
